gulpfile.babel.js

```diff
@@ -136,10 +136,13 @@ gulp.task('build_client', () => {
   webpack({
     watch: true,
     context: path.resolve(__dirname, 'client'),
-    entry: './app.js',
+    entry: {
+      bundle : './app.js',
+      sw : './push/push_manager.js'
```

Comment:
Replace this line with `sw : './service-worker.js'`
Also, you should remove 134 line.